genai-on-vertex-ai/natural_language_to_sql/natural_language_to_sql.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<p>Is this really a prompt design tip? This is more of a BigQuery best practice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Yes, this is more of BQ/SQL table design practice, it's to reference to the to-be published blog / document as an example.